### PR TITLE
Conform library files to our code style

### PR DESCRIPTION
**Pull request by bennothommo**
In preparation for rolling out code style checks with GitHub Actions.

### CODE REVIEW

File: tests/Parse/IniTest.php [hunk 1]
```diff
@@ -244,7 +244,7 @@ public function testMultilinesValues()
         $vars = [
             'var' => "\\Test\\Path\\",
             'editorContent' =>
-'<p>Some
+        '<p>Some
```
Comment (LukeTowers) [inline tests/Parse/IniTest.php]: What's the reason for this change?

File: tests/Support/EmitterTest.php [hunk 2]
```diff
@@ -102,17 +110,35 @@ public function testBindPriority()
...
+        }, 40);
+        $emitter->bindEvent('event.test', function () use (&$result) {
+            $result .= 'the ';
+        }, 30);
+        $emitter->bindEvent('event.test', function () use (&$result) {
+            $result .= 'lazy ';
+        }, 20);
+        $emitter->bindEvent('event.test', function () use (&$result) {
+            $result .= 'dog';
+        }, 10);
```
Comment (LukeTowers) [inline tests/Support/EmitterTest.php]: hmm, the earlier version was definitely easier to read, is there anyway we can allow for this?

File: tests/fixtures/config/sample-config.php [hunk 3]
```diff
@@ -15,7 +15,7 @@
 
     'debug' => true,
 
-    "debugAgain"  =>  FALSE ,
+    "debugAgain"  =>  false ,
```
Comment (LukeTowers) [inline tests/fixtures/config/sample-config.php]:
```suggestion
    "debugAgain"  =>  false,
```
there's an extra space there.

Comment (bennothommo) [reply tests/fixtures/config/sample-config.php]: @LukeTowers I have a feeling that the spacing there is intentional, as it is here:
https://github.com/octobercms/library/blob/4099148a026b3d11091a76056811124db2c31aa7/tests/fixtures/config/sample-config.php#L54
I would imagine it's testing to see that the config values are still read correctly even when someone mangles the spacing.

File: tests/Parse/IniTest.php (hunk 1 above)
Comment (bennothommo) [reply tests/Parse/IniTest.php]: PHPCS sees the above as incorrect indentation. I've formatted the string content there to match the original spacing but also keep the correct indentation.

https://github.com/octobercms/library/pull/428/commits/c487ef37cbf84844c92aa46fc296ff69672ba93a

File: tests/fixtures/config/sample-config.php (hunk 3 above)
Comment (LukeTowers) [reply tests/fixtures/config/sample-config.php]: In that case we should leave it as FALSE and exclude this file from coding style checks @bennothommo

Comment (bennothommo) [reply tests/fixtures/config/sample-config.php]: Done :)

**Comment by LukeTowers** [comment]
@bennothommo is this good to go now?

**Comment by bennothommo** [comment]
@LukeTowers yep, good to go :)
